src/components/atoms/OcProgress/OcProgress.vue

```diff
@@ -57,12 +61,24 @@ export default {
         return value.match(/(primary|passive|success|warning|danger)/)
       },
     },
+    /**
+     * Determines if the progress bar should be displayed in an indeterminate state.
+     */
+    indeterminate: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
   },
 
   computed: {
-    $_ocProgress_classes() {
+    classes() {
       return `oc-progress oc-progress-${this.size} oc-progress-${this.variation}`
     },
+    progressValue() {
+      const num = (this.value / this.max) * 100
```

Comment:
If `max` can be `null` or `undefined` that edge case should be handled here. E.g. an early return like this:
```js
if(!this.max) {
  return '-'
}
```